feature-libs/cart/wish-list/core/store/actions/wish-list.action.ts

```diff
@@ -41,57 +39,29 @@ export class CreateWishListFail extends StateUtils.EntityFailAction {
 interface LoadWishListPayload {
   userId: string;
   /**
-   * Used to compute wishlist cart name and find it in list of all carts.
+   * This is tempCartId which is exact the wishlist name computed from customerId
    */
-  customerId: string;
-  /**
-   * When we try load wishlist for the first time we don't know cart id.
-   * Instead we create temporary cart with id equal to wishlist name to keep track of loading/error state.
-   */
-  tempCartId: string;
+  cartId: string;
 }
 
+/**
+ * When we try load wishlist for the first time we don't know cart id.
+ * Instead we create temporary cartId from wishlist name to keep track of loading/error state.
+ */
```

Comment:
the `cartId` is the temporary id. Before, it use `tempCartId` to hold the temp cartId, when `LoadWishListSuccess`, it will have the real cart id, and temp cart id both. Now, we only have one `cartId`. When start load, it is temp id. When load success, it is real cart id.